frontend/language/src/nb.json

```
@@ -981,6 +981,7 @@
   "settings_modal.left_nav_tab_policy": "Tilganger",
   "settings_modal.left_nav_tab_setup": "Oppsett",
   "settings_modal.loading_content": "Laster inn data",
+  "settings_modal.maskinporten_no_scopes_available": "Du har ingen scopes lagt til i Maskinporten, så du kan ikke velge scope her.",
```

What about something like this? 🤔 
```suggestion
  "settings_modal.maskinporten_no_scopes_available": "Du har ingen tilgjengelige scopes å velge mellom fordi din bruker ikke er tilknyttet noen scopes i Maskinporten.",
```